src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/SentReferralSummary.kt

```diff
@@ -59,12 +60,13 @@ class SentReferralSummary(
   @Id val id: UUID,
   @ElementCollection
   @CollectionTable(name = "referral_assignments")
+  @Where(clause = "superseded = false")
   val assignments: MutableList<ReferralAssignment> = mutableListOf(),
```

Comment:
@sldblog I have made this changes only to `SentReferralSummary` entity which is used only for viewing the dashboard and I believe it will affect the reloading scenario.